R/register_extendr.R

```diff
@@ -36,39 +36,54 @@ register_extendr <- function(path = ".", quiet = FALSE, force_wrappers = FALSE)
 
   outfile <- rprojroot::find_package_root_file("R", "extendr-wrappers.R", path = path)
 
-  # If force_wrappers is TRUE, use tryCatch() to generate minimal wrappers even
-  # when there's some error (e.g. the symbol cannot be found).
-  # If FALSE, execute make_wrappers() only when the package can be loaded.
+  # If force_wrappers is TRUE, generate minimal wrappers even when there's some
+  # error (e.g. the symbol cannot be found).
   if (isTRUE(force_wrappers)) {
-    tryCatch(
-      make_wrappers(pkg_name, pkg_name, outfile, use_symbols = TRUE, quiet = quiet),
-      error = function(...) {
-        warning(
-          "Generating the wrapper functions failed, so a minimal one is used instead",
-          call. = FALSE
-        )
-        make_example_wrappers(pkg_name, outfile)
-      }
-    )
-  } else if (requireNamespace(pkg_name, quietly = TRUE)) {
-    make_wrappers(pkg_name, pkg_name, outfile, use_symbols = TRUE, quiet = quiet)
+    error_handle <- function(e) {
+      msg <- "Generating the wrapper functions failed, so a minimal one is used instead"
```

Comment:
The word "one" doesn't refer to anything in this sentence. (It can't refer to "wrapper functions", as that is plural.) Maybe write "a minimal wrapper file"?